### PR TITLE
Code review groups: polish drag and drop areas

### PR DESCRIPTION
**Pull request by molly-moen**
Update the drag and drop behavior for code review groups to make it easier to deal with empty groups and the unassigned group. Previously empty groups only had a small target area for dragging and dropping a new member, which could be interpreted as not being able to add to the empty group. In addition, the unassigned panel only allowed dropping into the area where there were already unassigned members, or a small line at the top if there were no unassigned members. This cleans up this behavior to the following:
- you can always drag and drop members anywhere in the unassigned panel
- empty groups have a blank placeholder spot to make it clear where to drop new members.

Screen recording of new behavior:

[new behavior](https://user-images.githubusercontent.com/33666587/149429366-04045717-ded4-4c17-b32c-6ecb35852162.mp4)

## Links
- jira ticket: [JAVA-387](https://codedotorg.atlassian.net/browse/JAVA-387)

## Testing story
Tested locally

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/codeReviewGroups/StudentGroup.jsx [hunk 1]
```diff
@@ -20,18 +25,24 @@ export default function StudentGroup({droppableId, members}) {
...
+                  followerId={member.followerId}
+                  name={member.name}
+                  index={index}
+                  key={member.followerId}
+                />
+              ))}
+              {provided.placeholder}
+            </div>
+          ) : (
+            showEmptyGroupPlaceholder && <div style={styles.emptyGroup} />
```
Comment (bencodeorg) [inline apps/src/templates/codeReviewGroups/StudentGroup.jsx:44]: I guess I'm a little confused by the two separate props, is it not possible in both cases for each scenario (`CodeReviewGroup` and `UnassignedStudentsPanel`) to pass in their own `emptyGroupStyle` or something like that?

**Review by bencodeorg — COMMENTED** [review]
Little implementation question, but looks much better! 🎉

File: apps/src/templates/codeReviewGroups/StudentGroup.jsx (hunk 1 above)
Comment (molly-moen) [reply apps/src/templates/codeReviewGroups/StudentGroup.jsx:44]: The two props are doing slightly different things here.`dropAreaStyle` is for the entire area to drop into, which for unassigned we want to be as large as the panel and not have a border. We want this to be the case for both empty and non-empty instances of the unassigned panel. `showEmptyGroupPlaceholder` is whether or not to have a div for an empty group or not.

**Review by bencodeorg — APPROVED** [review]
LGTM!

**Review by sanchitmalhotra126 — APPROVED** [review]
Nice!

**Pull request merged**